docs/sources/static/_index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Static mode
+weight: 200
+---
+
+# Static mode
+
+Static mode is the original runtime mode of Grafana Agent. Static mode is
+composed of different _subsystems_:
```

Comment:
Do we want to mention that it uses the YAML configuration file so that people understand what we're talking about?

As a side note (but the answer is probably no) do we want to mention by name the things that _only_ static mode supports (scraping service, host filtering, dynamic config)?